images/win/toolsets/toolset-2016.json

```diff
@@ -226,6 +226,9 @@
             "Microsoft.VisualStudio.Component.VC.Runtimes.ARM64.Spectre",
             "Microsoft.VisualStudio.Component.Workflow",
             "Microsoft.VisualStudio.Workload.Office"
+        ],
+        "build_workloads": [
```

Comment:
`buildtools_workloads`

Reply:
Thank you for suggestions, @maxim-lobanov , I made all the proposed corrections, could you please check it? :)